sdk/resourcemanager/azure-resourcemanager-resources/src/main/java/com/azure/resourcemanager/resources/models/WhatIfOperationResult.java

```diff
@@ -17,16 +17,22 @@
 public interface WhatIfOperationResult extends HasInnerModel<WhatIfOperationResultInner> {
 
     /**
+     * Gets the status of the What-If operation.
+     *
      * @return status of the What-If operation.
      */
     String status();
 
     /**
+     * Gets list of resource changes predicted by What-If operation.
+     *
      * @return list of resource changes predicted by What-If operation.
      */
     List<WhatIfChange> changes();
 
     /**
+     * Gets error when What-If operation fails.
```

Comment:
```suggestion
     * Gets the error when What-If operation fails.
```

Reply:
Fixed in the [commit#c526df8](https://github.com/Azure/azure-sdk-for-java/pull/43192/commits/c526df8f1d4631f24b05aada35920868ce4d1535)